app/views/school_moves/index.html.erb

```diff
@@ -1,5 +1,15 @@
 <%= h1 t(".title"), size: "xl" %>
 
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-two-thirds">
+    <% t(".description").each do |paragraph| %>
```

Comment:
In general we've not been using locale strings for hard coded text. Mostly because we don't think it'll need to be translated and it's easier to follow if all the content is in one file (in this case the view). If you think it's better as locale strings then I'm happy with that approach too, just thought I'd highlight that's a little inconsistent.